export/policy.ts

```diff
@@ -0,0 +1,72 @@
+import { PhylumApi } from 'phylum';
+
+/**
+ * Helper function for fetching a policy from Phylum.
+ */
+async function fetchPolicyRequest(url: string): Promise<string> {
+    const response = await PhylumApi.fetch("v0/", url, {});
+    
+    if(!response.ok) {
+        console.error(`\nFailed to get the policy for project '${projectId}'`);
```

Comment:
Is `projectId` set here?

Reply:
no, that's a holdout from some other code. I'll clean that up!